datafusion/physical-expr/src/equivalence/class.rs

```diff
@@ -422,6 +423,32 @@ impl EquivalenceGroup {
         self.bridge_classes()
     }
 
+    #[allow(clippy::type_complexity)]
+    fn adjacent_set(&self) -> HashSet<(&Arc<dyn PhysicalExpr>, &Arc<dyn PhysicalExpr>)> {
```

Comment:
Can we please add documentation to these functions about what they do and what the returned structure represents?